client/Board.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+import Box from './Box';
+
+class Board extends Component {
+
+  render() {
+    let row0 = 0;
+    let row1 = 1;
+    let row2 = 2;
+    let row3 = 3;
+    let col0 = 0;
+    let col1 = 1;
+    let col2 = 2;
+    let col3 = 3;
+    console.log("toggle:",this.props.toggle);
+		return (
+
```

Comment:
We should map over the array to render boxes so we can extend the board in the future